source/game.programme.programmelicence.bmx

```diff
@@ -969,7 +969,18 @@ Type TProgrammeLicence Extends TBroadcastMaterialSource {_exposeToLua="selected"
 	'override
 	Method HasBroadcastTimeSlot:int()
 		if GetSubLicenceCount() = 0 and GetData()
-			If Super.HasBroadcastTimeSlotEnabled() 
+			'TODO überdenken; Ist es sinnvoll, Timeslots überhaupt für wiederverwendbare Programmdaten
+			'hinterlegen zu können oder sind die immer lizenzspezifisch
+			'denn mit dem Überschreiben der HasBroadcastTimeSlot-Methode
+			'müssten man auch die getter für die Slots anpassen
+			'Den Fall nur ein Slot gesetzt würde ich komplett ignorieren (Konfigurationsfehler)
```

Comment:
Der Grundgedanke war ja, dass es ein Programm X geben koennte - und dazu dann 2 Lizenzen mit verschiedenen Ausstrahlungszeiten (22-24 bzw ohne Beschraenkung).

Die Getters/Has** sind ja in TBroadcastMaterialSource(Base) definiert - und sowohl TProgramme als auch TProgrammeLicence erben von dieser Klasse (ja ... organisch unschoen gewachsen).

Reply:
Beim Testspielen ist mir in der so umgesetzten Variante noch nichts aufgefallen, was nicht funktioniert (unterschiedliche Slots für verschiedene Serienfolgen). Tatsächlich kommen ja wiederverwendete Programmdaten für unterschiedliche Lizenzen noch nicht vor. Deshalb habe ich das TODO als Gedankenstütze dringelassen.